packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -249,6 +249,12 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 					editor.insertContent(result.html);
 				} else if (cmd.name === 'editor.focus') {
 					editor.focus();
+				} else if (cmd.name === 'editor.execCommand') {
+					if (!('ui' in cmd.value)) cmd.value.ui = false;
+					if (!('value' in cmd.value)) cmd.value.value = null;
+					if (!('args' in cmd.value)) cmd.value.args = {};
+
+					editor.execCommand(cmd.value.name, cmd.value.ui, cmd.value.value, cmd.value.args);
```

Comment:
Thanks for updating this part as well, and I think the implementation makes sense.